### ISSUE TITLE
CascView not finding files

### ISSUE THREAD

**Issue by WainPetopia** [ladislav-zezula/CascLib#259]
CascView currently fails to find WoW files (see screenshot). It has the same issue with retail and ptr versions.

I'm not sure when the problem began, but maybe it's because CascView hasn't been rebuilt in a while. I see there are some recent commits to CascLib for 10.1.7 but the latest version of CascView seems to date to July, so it may simply not be up to date. Hopefully it's that simple :)

Thanks! 

![CascView Error](https://github.com/ladislav-zezula/CascLib/assets/25504585/5dde1692-7ec0-4d05-be1e-576555ff4683)

**Comment by ladislav-zezula** [comment]
Hi,

yeah, believe it or not, it is *that* simple :-)

Due to linker incompatibility, the build didn't go through and there was old version of CascView on my website. The picture in your original post is basically a download file, which is one of the file name sources for CascLib.

TLDR: Fixed.

**Comment by WainPetopia** [comment]
Awesome! Thanks for the quick fix :)
